content/news/speaker-support.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Speaker Support for PyLadiesCon 2024"
+meta_title: "Speaker Support for PyLadiesCon 2024"
+description: "We have speaker support for those of you who need a bit more reassurance and encouragement!
+"
+date: 2024-08-28T05:00:00Z
+categories: ["Blog Post",]
+author: "Cheuk"
+tags: ["python", "community", "conference", "public speaking", "pyladies"]
+draft: false
+layout: news
+---
+
+Since there are only 2 weeks until the [Call for Propocal](https://pretalx.com/pyladiescon-2024/cfp) closes, we do not want to you miss this speaking opportunity. Do you still need some encouragement in submitting a proposal? Maybe you need a bit more help? Don't worry! We have speaker support for those of you who need a bit more reassurance and encouragement!
+
+## Support for new speakers
+
+If you are new to public speaking and would love to get a friendly mentor to help you prepare your submission or your talk when you are accepted. Please contact us at [pyladiescon@pyladies.com](mailto:pyladiescon@pyladies.com) with the title **Looking for speaker mentor** We will try our best to find a mentor for you. All you need to do is tell us a bit about yourself (where you are based, what language you would like to speak with your mentor and what do you need help with) so that we know how we can meet your needs.
+
+## Offering your help
+
+Suppose you are an experienced speaker and would like to help mentor someone. In that case, you can also [email us](mailto:pyladiescon@pyladies.com) with the title **I want to be a speaker mentor** - tell us a bit about yourself (where you are based, what language you speak, and briefly describe your past speaking experience) so we can match you up with someone.
+
```

Comment:
Below are a list of experienced speakers who have offered to support and mentor aspiring PyLadiesCon speakers.

- Mariatta. [Book an appointment](https://mariatta.ca/posts/pyladiescon-speaker-support/) through her website.

Reply:
@nceder can I add your info here? Are you still interested in offering speaker support? What do you want to mention here/how can people contact you.

Reply:
Same as above, I think it is better to do the assignment and matching behind the scene to protect the mentors. We can also use a form to collect interested to be mentors

Reply:
I think @DennyPerez18 also said she can help with Spanish speakers.

Reply:
Ok so now we have @DennyPerez18 and @Mariatta, I am happy to do that as well so we have 3 confirmed. Is @nceder also confirmed?